src/execution/sql/thread_state_container.cpp

```diff
@@ -1,13 +1,13 @@
 #include "execution/sql/thread_state_container.h"
 
-#include <tbb/enumerable_thread_specific.h>
+#include <tbb/mutex.h>
```

Comment:
I don't think you need this header if you're using `common::SpinLatch`. Thinking again, why isn't the header for `common::SpinLatch` included here?